packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexContainer.tsx

```diff
@@ -52,9 +52,9 @@ const StyledContainer = styled.div`
 `;
 
 const StyledContainerWithPadding = styled.div`
-  height: calc(100% - 40px);
+  height: calc(100% - ${({ theme }) => theme.spacing(10)});
   margin-left: ${({ theme }) => theme.spacing(2)};
-  width: 100%;
+  width: calc(100% - ${({ theme }) => theme.spacing(2)});
```

Comment:
Could we use `box-sizing: border-box` instead of calculating width?

Reply:
I tried! It didn't work ;(

Reply:
@ehconitin could you try taking over this change?